elastic/apm-server
# aggregation/spanmetrics: rename metrics

## Motivation/summary

Add a `span.` prefix to service destination metrics fields, so they are consistent with existing `span.destination.service.*` fields. Also, add a system/approval test so we can visually inspect the documents.

## Checklist

~- [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~

I have considered changes for:
~- [ ] documentation~
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
~- [ ] telemetry~
~- [ ] Elasticsearch Service (https://cloud.elastic.co)~
~- [ ] Elastic Cloud Enterprise (https://www.elastic.co/products/ece)~
~- [ ] Elastic Cloud on Kubernetes (https://www.elastic.co/elastic-cloud-kubernetes)~

## How to test these changes

1. send some spans with a service destination resource
2. wait for aggregation interval
3. look for service destination metrics docs, the new fields should be grouped with existing service map fields (span.service.destination.*)

## Related issues

None.
